appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/property/impl/BackgroundCorner.kt

```diff
@@ -0,0 +1,56 @@
+package com.bumble.appyx.interactions.core.ui.property.impl
+
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.AnimationVector1D
+import androidx.compose.animation.core.Easing
+import androidx.compose.animation.core.VectorConverter
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.clip
+import com.bumble.appyx.interactions.core.ui.context.UiContext
+import com.bumble.appyx.interactions.core.ui.math.lerpFloat
+import com.bumble.appyx.interactions.core.ui.property.Interpolatable
+import com.bumble.appyx.interactions.core.ui.property.MotionProperty
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+class BackgroundCorner(
```

Comment:
FYI We already have one called `RoundedCorners`

Reply:
<img width="300" alt="Screenshot 2023-06-23 at 21 39 55" src="https://github.com/bumble-tech/appyx/assets/10270046/07413706-8bf9-4c1e-9e02-418825aeb0a3">

Reply:
Ah! Just didn't update project in time :)